Mathlib/FieldTheory/Separable.lean

```diff
@@ -555,6 +561,14 @@ variable (F K : Type*) [CommRing F] [Ring K] [Algebra F K]
 -- Note that right now a Galois extension (class `IsGalois`) is defined to be an extension which
 -- is separable and normal, so if the definition of separable changes here at some point
 -- to allow non-algebraic extensions, then the definition of `IsGalois` must also be changed.
+
+/--
+An element `x` of an algebra `A` over a commutative ring `R` is said to be *separable*, if its
+minimal polynamial over `R` is separable. Note that the minimal polynomial of any element not
+integral over `R` is defined to be `0`, which is not a separable polynomial.
+-/
+notation " IsSeparable " F:max x:max => (Polynomial.Separable (minpoly F x))
```

Comment:
I personally think this deserves to be a def. Among other things, it unlocks dot notation.

Reply:
Good point, let's make it a `def`.

Reply:
Good! I am working on this. I am also trying to decompose many theorems in `Algebra.IsSeparable` into a theorem of `IsSeparable` and a theorem using the previous one directly. This takes more time that I thought.

Reply:
What you can do is to open another PRs. Here you are just renaming stuff, right? I think it is better to not do anything else in this PR.

Reply:
I see. Thank you!